awx/ui/src/components/Schedule/shared/DateTimePicker.js

```diff
@@ -33,7 +33,11 @@ function DateTimePicker({ dateFieldName, timeFieldName, label }) {
 
   return (
     <FormGroup
-      fieldId={`schedule-${label}-datetime`}
+      fieldId={
+        dateFieldName.startsWith('start')
```

Comment:
This change relies on the dateFieldName  to always contain `start` or `end` as started string, but dateFieldName is just a string that can be modified to be any.  Did I misread this change?

Reply:
I initially implemented this change in order to get the tests to pass, however, I will revert this change and introduce a dataCy prop that we can use as a selector for our unit tests.